src/openapiGenerator/mockGenerator/formatResponse.ts

```diff
@@ -35,10 +35,38 @@ export const extractResponses = (obj: OpenAPI.Document): ResponsesType => {
                 } else if (response.content && response.content[APPLICATION_JSON]?.schema) {
                     mock = response.content[APPLICATION_JSON].schema;
                 }
-                mock = typeof(mock) === "string" || isExample ? mock : generateData("", mock, modelSchemas);
+                mock = typeof (mock) === "string" || isExample ? mock : generateData("", mock, modelSchemas);
```

Comment:
this should be `typeof(mock)` or `typeof mock`

![image](https://user-images.githubusercontent.com/49055952/91013985-2f486300-e61b-11ea-8c45-0b4bdd2265ed.png)

Reply:
changed to typeof operand